src/components/AllMovies/AllMovies.jsx

```diff
@@ -19,3 +19,8 @@ export default function AllMovies(props) {
 
   return <main className='all-movies-container'>{movieCards}</main>
 }
+
+AllMovies.propTypes = {
+  movies: PropTypes.array.isRequired,
```

Comment:
To really get the benefit of proptypes, could you go more indepth here so that it is typechecking not just for an array, but that the contents of that array are what you'd expect?